mongodb.md

```diff
@@ -76,10 +75,10 @@ Within this file, you can add a `mongodb` connection using the `mongodb` driver:
 
 Once your configuration is complete, you can use the `mongodb` connection in your application:
 
-- [Using Eloquent](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/), models can be stored in MongoDB collections. In addition to the standard Eloquent features, the Laravel MongoDB package provides additional features such as embedded relationships. Using raw SQL queries is not supported obviously, as MongoDB is a NoSQL database.
+- [Using Eloquent](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/), models can be stored in MongoDB collections. In addition to the standard Eloquent features, the Laravel MongoDB package provides additional features such as embedded relationships. Raw SQL queries is not supported but advanced MongoDB queries can be used.
 - [Write complex queries](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/query-builder/) using the query builder.
-- [Using the Cache](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/cache/), the `mongodb` cache driver is optimized to use MongoDB features such as TTL indexes to automatically clear expired cache entries.
-- Using the Session with the `cache` session driver.
+- [Using the Cache](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/cache/), the `mongodb` Cache driver is optimized to use MongoDB features such as TTL indexes to automatically clear expired cache entries.
+- Using the Session with Laravel's `cache` and the `mongodb` cache driver. 
 - [Using the Queue](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/queues/) with the `mongodb` queue driver.
-- [Storing files in GridFS](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/gridfs/), thanks to an integration of the  [GridFS Adapter for Flysystem](https://flysystem.thephpleague.com/docs/adapter/gridfs/).
-- And third party packages using a Database connection can be used with MongoDB, please [raise an issue](https://github.com/mongodb/laravel-mongodb/issues/new/choose) for unsupported packages you would like to use.
\ No newline at end of file
+- [Storing files in GridFS](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/gridfs/), by way of the [GridFS Adapter for Flysystem](https://flysystem.thephpleague.com/docs/adapter/gridfs/).
+- And third party packages using a database connection can be used with MongoDB.
```

Comment:
```suggestion
- Third party packages using a database connection can be used with MongoDB.
```

I don't think we need "And" for a final bullet. Consider adding "Most", though. I imagine there's some concern in misleading users to think that any and all third party packages will work out of the box with a `mongodb` database connection.